.github/workflows/update-labels.yaml

```diff
@@ -7,12 +7,11 @@ on:
   # If an upstream workflow if completed, get only the artifacts from that workflow, and update labels
   workflow_run:
     workflows:
-      [
-        "ARM Auto SignOff",
-        "SDK Breaking Change Labels",
-        "SDK Suppressions",
-        "TypeSpec Requirement",
-      ]
+      - "ARM Auto SignOff"
+      - "SDK Breaking Change Labels"
+      - "SDK Suppressions"
+      - "TypeSpec Requirement"
+      - "TEST-IGNORE Swagger BreakingChange - Analyze Code"
```

Comment:
This change will be reverted before this PR is merged as we don't want to impact current breaking change check in the unified pipeline.